README.md

```diff
@@ -18,12 +18,37 @@ We welcome contributions and feedback. Please follow our [contributing guideline
 
 ---
 
-## To run locally
+## one-time setup
+- clone this repo
+- run **npm install**
 
-- Clone this repository
+## To run locally
 - Ensure that you are at the solution folder
-  - **npm install**
-  - **npm run gulp-trust-dev-cert**
+  - run **npm run gulp-trust-dev-cert**
   - Replace `REPLACE_WITH_YOUR_SHAREPOINT_SITE` in serve.json with your sharepoint site
-  - **npm run serve**
+  - run **npm run serve**
+    - this should open up a window in the browser called a SharePoint workbench. 
   - To use a.clouddev.laserfiche.com: Open browser dev tools and go to site Local Storage: set 'spDevMode' to true
+
+## To build solution for development/testing changes
+- **npm run build**
+- **npm run package**
+- this should result in the creation of a file with the path `/sharepoint/solution/laserfiche-sharepoint-integration.sppkg` from the root folder.
+- reference the [Admin Documentation](https://laserfiche.github.io/laserfiche-sharepoint-integration/) for instructions on how to use the solution file to test your changes to the WebParts in SharePoint Sites.
+
+## To build solution for production
+- **npm run bundle-prod**
```

Comment:
To test in a SharePoint site with files hosted on SharePoint site
- Note: this could be used as a production package